src/complex.rs

```diff
@@ -132,6 +133,38 @@ impl<T: Clone + Float> Complex<T> {
         let (r, theta) = self.to_polar();
         Complex::from_polar(&(r.sqrt()), &(theta/two))
     }
+    
+    /// Raises a number to a floating point power.
+    #[inline]
+    pub fn powf(&self, value: T) -> Complex<T> {
+        let (r, theta) = self.to_polar();
+        Complex::from_polar(&(r.powf(value)), &(theta*value))
+    }
+    
+    /// Returns the logarithm of the number with respect to an arbitrary base.
+    #[inline]
+    pub fn log(&self, value: T) -> Complex<T> {
+        let (r, theta) = self.to_polar();
+        // Comment for code review:
+        // There is for sure a faster way to get the constant, but I don't know how to do that in this generic context 
+        // Any hints?
+        let e = T::one().exp(); 
```

Comment:
Very right, the arguments shouldn't be called `value`. Looking through the current API I recognized that the different `pow*` methods use different names for their arguments:
- `PrimInt: pow(self, exp: u32)`
- `Float: powi(self, n: i32), powf(self, n: T)`
- `Rational: pow(&self, expon: i32)`

From this my choice would be `exp`. We could also use this opportunity to clean up the naming and pick one parameter name for all implementations of `pow*`. Would this be a good idea? Parameter names in Rust are from my understanding not part of the method signature and therefore this wouldn't be a breaking chance.